account_bank_statement_import_auto_reconcile/models/account_bank_statement_import_auto_reconcile_odoo.py

```diff
@@ -12,7 +12,8 @@ class AccountBankStatementImportAutoReconcileOdoo(models.AbstractModel):
     @api.multi
     def reconcile(self, statement_line):
         """Find an open invoice for the statement line's partner"""
-        matches = statement_line.get_reconciliation_proposition(statement_line)
+        matches = statement_line.get_reconciliation_proposition(
+            statement_line.ids)
```

Comment:
read the definition of the function: https://github.com/OCA/OCB/blob/10.0/addons/account/models/account_bank_statement.py#L617 (you shouldn't pass anything at all here)